index.js

```diff
@@ -91,6 +102,8 @@ function magnetURIDecode (uri) {
     result.urlList = result.urlList.concat(result.ws)
   }
 
+  if (result.x && result.x.pe) result.peerAddress = result.x.pe
```

Comment:
And use the name `peerAddresses`, since with the change above will be always an array.

Reply:
Change done! 👍